packages/atomic/src/components/facets/atomic-category-facet/atomic-category-facet.pcss

```diff
@@ -1,6 +1,3 @@
 @import "tailwindcss/base";
 @import "tailwindcss/utilities";
-
-:host {
-  display: block;
-}
+@import "../facet/base-facet.pcss";
```

Comment:
There's no way to import a css file into a functional component, so each base component would need to import the styles themselves.